src/commands/doctor.ts

```diff
@@ -95,9 +95,24 @@ module.exports = {
       expoVersion = "-"
       expoWorkflow = "not installed"
     }
-
     const expoInfo = [column1("expo"), column2(expoVersion), column3(expoWorkflow)]
 
+    let depExpoCliInfo = null
+
+    try {
+      const expoCliVersionOutput = await run("expo-cli --version", { trim: true })
+      if (expoCliVersionOutput) {
+        const expoCliVersion = expoCliVersionOutput.replace("v", "")
+        depExpoCliInfo = [
+          column1("expo global cli"),
+          column2(expoCliVersion),
+          colors.yellow("Deprecated: Found 'expo-cli' installed. Please remove it."),
```

Comment:
I think you still need `column3` here

`column3(colors.yellow(.....`

Reply:
ahhh thought i did one sec will do so now.

Reply:
so embarrassed totally did this already just forgot to push